packer/cache_config_unix.go

```diff
@@ -0,0 +1,20 @@
+// +build darwin freebsd linux netbsd openbsd solaris
+
+package packer
+
+import (
+	"os"
+	"path/filepath"
+)
+
+func getDefaultCacheDir() string {
+	var defaultConfigFileDir string
+
+	if xdgConfigHome := os.Getenv("XDG_CONFIG_HOME"); xdgConfigHome != "" {
```

Comment:
This is bad, not is `XDG_CONFIG_HOME` is `XDG_CACHE_HOME`

Reply:
Ah let me fix.

Reply:
I have pushed update to fix this